pkg/handlers/healthProbes.go

```diff
@@ -27,8 +31,21 @@ func LivenessProbe(w http.ResponseWriter, r *http.Request) {
 
 // ReadinessProbe checks if Redis is available.
 func ReadinessProbe(w http.ResponseWriter, r *http.Request) {
-	glog.V(2).Info("readinessProbe - Checking Redis connection.")
 
+	glog.V(2).Info("readinessProbe - Checking Redis connection.")
+	ns := os.Getenv("POD_NAMESPACE")
+	if ns == "" {
+		ns = "open-cluster-management"
+	}
+	options := metav1.GetOptions{}
+	_, err := config.GetKubeClient().CoreV1().Pods(ns).Get("search-redisgraph-0", options)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			glog.Info("Redisgraph pod is not present - will re-check once it is enabled. Err:", err)
+			return
```

Comment:
If we return here, it will never send a response and the readinessProbe will timeout.